charts/jenkins/pod-templates/nuxeo-benchmark-lts-2021.yaml.gotmpl

```diff
@@ -32,7 +32,7 @@
     ttyEnabled: {{ .Values.podTemplates.ttyEnabled }}
     workingDir: {{ .Values.podTemplates.workingDir }}
   - args: ""
-    command: ""
+    command: "redis-server /usr/local/etc/redis/redis.conf"
```

Comment:
Do you need to alter the command? According to the [Configuration](https://github.com/bitnami/containers/tree/main/bitnami/redis#configuration-file) doc, mounting the conf file should be enough.

Reply:
Sorry, I got confused as the repo only displays a main branch and the last commit was  "[bitnami/redis] Release 7.2.4-debian-12-r9". So I quickly jumped to conclusion... But looking it up in the 6.2 folder it is used indeed:
https://github.com/bitnami/containers/blob/23b5d6ecad3adc425b25b7a751c760e95678cae2/bitnami/redis/6.2/debian-12/rootfs/opt/bitnami/scripts/redis-env.sh#L31

Looking at the configuration, the path is a bit different. I need to change `etc` for `mounted-etc` in my PR if I'm correct ? 
And then no need for the command override.

Reply:
I think the path would be `/opt/bitnami/redis/mounted-etc/redis.con`, as stated in the Configuration doc.
But, IMO, if the environment variable works, it is simpler than a configuration file!

Reply:
It doesn't seem to be enough indeed.
So I will mount the new configMap on the default path so I don't have to override the command:
`/opt/bitnami/redis/mounted-etc/redis.conf`
Getting there 😄